lib/puppet/type/openldap_schema.rb

```diff
@@ -11,4 +11,9 @@
     desc "The location to the schema file."
   end
 
+  newparam(:converttoldif, :boolean => true) do
+    desc "Convert legacy schema file to ldif format"
+    defaultto :true
```

Comment:
Couldn't we default to false if `path` ends with `.ldif` and true otherwise so that we don't have to set it explicitly?